.github/workflows/quick-merge.yml

```
@@ -0,0 +1,170 @@
+name: Quick Merge
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: false
+
+on:
+  pull_request_review:
+    types: [submitted]
+    branches: [main, develop]
```

actionlint (1.7.8)

"branches" filter is not available for pull_request_review event. it is only for merge_group, push, pull_request, pull_request_target, workflow_run events

(events)